### DISCUSSION TITLE
FNIRSI GC-01: Battery question and suggestions

### DISCUSSION THREAD

**Discussion by Vteriaza** [Gissio/radpro#182] [Q&A]
Hello, I'm new to forums. Greetings to all.
I have a fnirsi gc-01 and I'm using this new firmware, which I'm really liking.
I changed the battery from 1100ma to a 3000ma battery for my gc-01, with the factory battery it lasted about 12 or 13 hours working, when I put the 3000ma it lasted about 24 hours.
I installed the firmware two days ago and this testing the autonomy of the gc-01 and it has been in use for 43 hours, with conservative configuration.
The battery indicator does not work according to the battery capacity. According to the battery indicator, it ran out after 24 hours or so and now it is still working but I don't have data on what's left of the battery, it's an uncertainty.
And if any suggestion is accepted, I would miss a text-style screen where you can see at a glance several parameters, such as instantaneous dose, average, cumulative and some maximum, since I consider this firmware beautiful and elegant, but too minimalist. I also echo the original screen or original firmware, at the top and bottom ends where it says current date and current time at the top and begindate and lifetime at the bottom. I consider important data. In the latest FirmaWare update I don't see the date anywhere.
Thank you and greetings.

**Comment by Gissio** [comment]
Moving to discussions as this is not an issue (#32).

**Reply by Gissio** [reply]
Some users measured the discharge curve from the factory FNIRSI GC-01 here #28 (scroll to the bottom), which Rad Pro uses to calculate the battery level.

It is quite likely that your battery has a different discharge profile.

Thanks for the suggestions.

**Comment by Vteriaza** [comment]
Hola Gissio. muchas gracias por tu respuesta.
Bueno, lo primero es que no tengo ni idea de programacion ni de electronica,todo esto es simple curiosidad, y lamento no poder aportar nada al proyecto.
Estoy haceindo pruebas y como puse arriba,la bateria de 3000ma me duro 44 horas mas o menos, con el brillo de pantalla bajo y que se apagara en 30 segundos. 
Esta ultima comprobacion,la he hecho con brillo de pantalla en medio nivel y con la pantalla encendida siempre y le queda poca bateria teniendo en cuenta el voltaje de la misma. ahora mismo lleva encendido 31 horas 33 minutos y quedan 3.091 volts . se apagara en breves.
Una conclusion precipitada me hace ver que la pantalla no consume tanta energia? 
En la prueba primera, se paso el 90% del tiempo con la pantalla apagada y duro unas 10 u 11 horas mas, por lo que hay un gasto de energia que es mayor que el de la pantalla.?
Me gustaria dejar varias propuestas.
El grafico, en mi dispositivo no se ajusta a la realidad. Es decir, me esta dando una medida media de 0.112 microsieverts y segun el grafico me da una medicion entre 0.5 y superior. Por otra parte, el grafico no se conserva cuando apagas el dispositivo y lo vuelves a encender, en cuanto el averge si que se mantiene.
Sobre el tema de la pantalla ,estaria muy bien tener una opcion de atenuacion de pantalla seleccionancio un tiempo y luego el apagado de la misma a otro tiempo concreto. Eso podria hacer que alargase mas la duracion de la bateria.
Sobre la alarma,el firmware original, la alarma era mas alarmante, y se podia mutear mientras hacias una prueba, y ahora veo una alarma discreta y que no puedo pararla y suena siempre.
Sobre la lectura average,veo que lleva mucho tiempo para hacer una media, aunque manualmente puedo resetear cuando quiera , que esta muy bien y me gusta mucho, pero deberia de haber algun modo se sacara la media de cada minuto,por ejemplo.
Sigo con la prueba de la batteria mientras escribo, y llendo al hilo que me enviaste sobre la bateria, [martin-machacek](https://github.com/martin-machacek)  se le apago con 3.042v y en estos momentos mi dispositivo sigue funcionando con un voltaje de 2.834 y bajando rapidamente pero aun esta en marcha.
Tengo el detector Fnirsi GC-01 hace dos años, y hace solo unos dias que encontre esta maravilla de firmare y las posibilidades de este dispositivo. Mi pregunta como novato seria, en que puedo notar o como me puedes explicar , la diferencia de precision entre el firmware original y este ultimo, que ha cambiado? sin tener en cuenta su gran posibilidad de opciones y configuraciones.  hablo de fiabilidad y precision.
Como puse anteriormente, echo de menos una pantalla con mas datos de un solo vistazo, pantalla estilo texto con varios valores.
Como decia antes ,siento no poder aportar nada mas que ideas que quizas no sean posibles. Agradezco tu trabajo y el de los colaboradores por este proyecto tan interesante. 
Saludos. 
pd. el disositivo se apago 2.600v .

**Reply by Vteriaza** [reply]
Me he dado cuenta y creo que lo de la interpretacion del grafico ha sido un fallo mio. Si que mide bien. el fallo fue al leer la escala, que no me quedo muy claro.
